src/App.css

```diff
@@ -242,3 +243,215 @@ form h3 {
   padding: 100px 100px 150px;
   width: 100%;
 }
+
+@media (max-width: 768px) {
+  nav {
+    width: 100%;
+  }
+
+  .nav-menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-menu p {
+    font-size: 0.875rem;
+    color: #0290ff;
+    font-weight: bold;
+  }
+
+  .nav-menu ul {
+    list-style: none;
+    display: flex;
+    gap: 20px;
+    margin-left: 20px;
+  }
+
+  .nav-menu ul li a {
+    font-size: 0.413rem;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.9px;
+    color: #121212;
+  }
+
+  .userIconCont {
+    display: none;
+  }
+
+  .booksContainer {
+    padding: 50px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  form h3 {
+    font-size: 1rem;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: -0.18px;
+    color: #888;
+  }
+
+  .form-control {
```

Comment:
- [ ] Please try to make the form for adding book take the same with as the list of books as asked by the previous reviewer.

| Your design | Zeplin design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/191824839-7006e62e-4d18-46c7-a7b4-5948b719da7f.png) | ![image](https://user-images.githubusercontent.com/62290545/191825222-f016d3da-1a46-45cc-b446-d6d2b765327c.png) |

Reply:
@ShadyShawkat ,  Please assist to check the updated code before I request for the last review